### ISSUE TITLE
MBConvBlockWithoutDepthwise stride implemented in 1x1 projection, wasting expansion arithmetic

### ISSUE THREAD

**Issue by andravin** [tensorflow/tpu#660]
`MBConvBlockWithoutDepthwise` implements stride in the `1x1` projection convolution. When stride=2, the projection discards 3/4ths of the activations produced by the expansion. It would be equivalent to implement stride on the `3x3` expansion convolution instead, and this would reduce the total block arithmetic almost by a factor of 4.

https://github.com/tensorflow/tpu/blob/8462d083dd89489a79e3200bcc8d4063bf362186/models/official/efficientnet/efficientnet_model.py#L422-L442

**Comment by andravin** [comment]
The reduction in OPs is substantial: moving the stride from the projection convolution to the expansion reduces the total number of operations in EfficientEdgeTPU-S by about 24%.

**Comment by andravin** [comment]
At this point, it should be apparent that Google has no intention of responding to this issue. I will nevertheless leave this comment here in case it helps anyone who should stumble upon it:

Google actually fixed this design flaw in the updated implementation of the MBConWithoutDepthwise as it is used in MobileDets. In that work, they renamed the block "Fused convolution layer."

The implementation can be found here: https://github.com/tensorflow/models/blob/2986bcafb9eaa8fed4d78f17a04c4c5afc8f6691/research/object_detection/models/ssd_mobiledet_feature_extractor.py#L142-L147

Notice that the stride is now implemented in the expansion convolution:
```
    h = _conv(h, expanded_filters, kernel_size, strides=strides,
              activation_fn=activation_fn)
    if use_se:
      hidden_dim = _scale_filters(expanded_filters, 0.25)
      h = _squeeze_and_excite(h, hidden_dim, activation_fn=activation_fn)
    h = _conv(h, filters, 1, activation_fn=tf.identity)
```

**Comment by mingxingtan** [comment]
Hi @andravin   Thanks for the great point.  I have prepared a fix change internally, which should go out in the next release.

**Comment by allenwang28** [comment]
Fixed in https://github.com/tensorflow/tpu/pull/880/commits/32572cbc7458bded9a1551ae7984f76de8d69866

**Comment by yuezha01** [comment]
I noticed exactly the same issue recently when I looked into the Efficientnet_EdgeTPU repo. Thank @andravin for pointing out the issue here. 
Quick note here in case anyone uses the TFlite files (e.g. EfficientNet-EdgeTPU-S)  published in the[ efficientnet_edgeTPU repo]([url](https://github.com/tensorflow/tpu/tree/master/models/official/efficientnet/edgetpu)). Although the issue has been fixed in the new released code, the Tflite files still has stride of 2 in the point wise layer of fused block.
